frontend_tests/node_tests/compose.js

```diff
@@ -43,6 +48,70 @@ people.initialize_current_user(me.user_id);
 people.add(alice);
 people.add(bob);
 
+i18n.init({
```

Comment:
Actually I initially did something similar to that.
(To be precise
```
set_global('i18n', {
    t: function (s) { return s; },
});
```)
But then I ran into testing stuff where context was being passed to i18n, so I used this way.